python/cuml/preprocessing/model_selection.py

```diff
@@ -37,8 +26,8 @@ def train_test_split(
     seed: int = None,
 ) -> Tuple[cudf.DataFrame, cudf.DataFrame, cudf.DataFrame, cudf.DataFrame]:
     """
-    Partitions the data into four collated dataframes, mimicing sklearn's
-    `train_test_split`
+    Partitions cuDF data into four collated dataframes, mimicking
```

Comment:
It would be a good idea to open an issue for supporting cupy and numba arrays as well. I recently tried to use this and was surprised to find it only worked with cuDF.

Reply:
Oops. I just missed something quite important- the sklearn version takes a RandomState, not a seed. One further thing I noticed when trying to use this recently is that it's missing the test_size argument.